src/components/ActionSheet/ActionSheet.tsx

```diff
@@ -61,7 +61,7 @@ export const ActionSheet: React.FC<ActionSheetProps> = ({
   };
 
   if (process.env.NODE_ENV === "development" && !restProps.onClose) {
-    warn("can't close on outer click without onClose", "error");
+    warn("Требуется передать onClose для обработки закрытия изнутри", "error");
```

Comment:
Тут скорее про то, что по клику снаружи нельзя закрыть.
```suggestion
    warn('Без "onClose" нельзя закрыть ActionSheet по клику вне компонента', "error");
```